Three seconds is better

kheemchandra opened the pull request:
Studies have shown that 3 seconds is a far better LED delay than 1 second.

https://www.google.com

ashudhanik commented on blink.ino:
```diff
@@ -18,7 +18,7 @@ void setup() {
+  delay(3000);               // wait for a second
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
-  delay(2000);               // wait for a second
+  delay(3000);               // wait for a second
```
I believe it would be better if the light was off for 4 seconds and on for just 3.

ashudhanik commented:
If you make that change I'll be happy to merge this.

kheemchandra commented:
I changed off time to 4 seconds and on time to 2 seconds  and also removed some trailing whitespace that I found. Anything else you would like me to do ?

ashudhanik commented:
ok thanks
